src/v3/obfuscate.test.ts

```diff
@@ -0,0 +1,180 @@
+import { obfuscateDocument, obfuscateData } from "./obfuscate";
+import { OpenAttestationCredentialWithInnerIssuer, validateSchema, verifySignature, wrapDocument } from "../";
+import { get } from "lodash";
+
+import { SchemaId } from "../shared/@types/document";
+import { IdentityProofType, Method, OaProofType, TemplateType } from "../../src/__generated__/schemaV3";
+import { toBuffer } from "../shared/utils";
+
+const openAttestationData: OpenAttestationCredentialWithInnerIssuer = {
```

Comment:
I think what would be nice for those test is to directly use the expected object as output 

```typescript
const obj = {
  key1: "value",
  key2: "value"
}
const {data} = obduscate(obj, "key1")
expect(data).toStrictEqual({ key2:value})
```

I would suggest to try with the smallest open-attestation credential doable

Reply:
not done ? reason ?

Reply:
Removed template from sample document to make the test data smaller but am unable to test directly with expected output as the salt hash is non-deterministic...V2 was able to test obfuscateData in that manner because the method removed attributes from data: any whereas in V3 obfuscateData requires data: OpenAttestationVerifiableCredential which undergoes wrapping first. Or would you like me to omit the fields from the wrapped data and decode, remove the salt, encode it back and expect the resultant OpenAttestationVerifiableCredential?

Reply:
will check later =)

Reply:
okay!